.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Documentation
+about: Update the documentation
+title: 'Documentation: '
+labels: 'alert: NEED ACCOUNT KEY, alert: NEED MORE DEFINITION, alert: NEED CYCLE ASSIGNMENT, type: documentation'
+assignees: ''
+
+---
+
+*Replace italics below with details for this issue.*
+
+## Describe the Task ##
+*Provide a description of the task here.*
+
+### Time Estimate ###
+*Estimate the amount of work required here.*
+*Issues should represent approximately 1 to 3 days of work.*
+
+### Sub-Issues ###
+Consider breaking the task down into sub-issues.
+- [ ] *Add a checkbox for each sub-issue here.*
+
+### Relevant Deadlines ###
+*List relevant project deadlines here or state NONE.*
+
+### Funding Source ###
+*Define the source of funding and account keys here or state NONE.*
+
+## Define the Metadata ##
+
+### Assignee ###
+- [ ] Select **engineer(s)** or **no engineer** required
+- [ ] Select **scientist(s)** or **no scientist** required
+
+### Labels ###
+- [ ] Review default **alert** labels
+- [ ] Select **component(s)**
+- [ ] Select **priority**
+- [ ] Select **requestor(s)**
+
+### Milestone and Projects ###
+- [ ] Select **Milestone** as a **METplus-Wrappers-X.Y.Z** version, **Consider for Next Release**, or **Backlog of Development Ideas**
+- [ ] For a **METplus-Wrappers-X.Y.Z** version, select the **METplus-Wrappers-X.Y.Z Development** project
+
+## Define Related Issue(s) ##
+Consider the impact to the other METplus components.
+- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)
+
+## Task Checklist ##
```

Comment:
This checklist is probably too detailed for doc updates. But perhaps keeping it consistent across templates is more useful than make it different in subtle ways?

Reply:
There could be instances where doc changes should be made in other repos, so I don't think it's necessary to remove this step.